routes/notification.js

```diff
@@ -0,0 +1,75 @@
+module.exports.notification = async (req, res) => {
```

Comment:
Use https://github.com/pugjs/pug to generate dynamic html pages for notifiactions